kpatch-build/create-diff-object.c

```diff
@@ -760,11 +760,39 @@ static void kpatch_compare_symbols(struct list_head *symlist)
 	}
 }
 
+#define CORRELATE_ELEMENTS(e1, e2)				\
```

Comment:
nit: not a big deal, but wouldn't the compiler figure out if inlining is a win here?

Reply:
I didn't go for a macro for inlining here but because I can use the macro for both `struct symbol *` and `struct section *` expressions.

If that's not considered good practice, I can have two static functions, but their code will be pretty much identical.

Reply:
Ah, I see.  I don't have a strong preference either way then.